src/components/ErrorBoundary.jsx

```diff
@@ -3,21 +3,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import postErrorToSlack from 'utils/postErrorToSlack';
 import { connect } from 'react-redux';
-import Error from 'pages/_error';
 
 // Implementation of https://reactjs.org/docs/error-boundaries.html
 // Using React class because it's not yet supported for functional components
 // according to https://stackoverflow.com/a/68075800/1940886
 class ErrorBoundary extends React.Component {
+  // The parent constructor needs to be called so that the
+  // proper functions are setup and views can be rendered
+  // eslint-disable-next-line no-useless-constructor
```

Comment:
afaik https://eslint.org/docs/rules/no-useless-constructor this means that by default, when a class inherits another one, it will call `super(props)` if there is no constructor, so the linter seems to be correct and the call is unneeded. For reference the following works:

```
class A {
    constructor (value) {
        this.value = value
    }

    print() { 
        console.log(this.value)
    }
}

class B extends A {
}
const b = new B('a')

b.print() # prints 'a'
```

Reply:
I tried removing the constructor, but it doesn't work. I found this blog which talks about this: https://overreacted.io/why-do-we-write-super-props/. Calling `super(props)` is required to set up the `this` variable, which might internally be used by React to function.